environment.yml

```diff
@@ -0,0 +1,16 @@
+name: rtdev
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python>=3.7
+  - numpy
+  - scipy
```

Comment:
Since this is just the "dev" environment I figured it would be nice to have around. Same with `pandas` and all the pre-commit and testing tools.

Also, github has an "insert suggestion" button that you can use to, e.g. suggest lines of code to be deleted (such as the `scipy` dependency).